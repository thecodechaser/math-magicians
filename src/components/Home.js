import React from "react";

const Home =()=>{
    return (
        <main>
            <h1>Welcome to our page!</h1>
            <p>End of the day, life itself is a math problem. Everything can be explained through math, 
                including the rotation of our Earth or creation of 
                the human kind. You will also find some inspirational math quotes for students with donuts</p>
            <p>There are certain calculations I should like to make with you,
To be sure that your deductions will be logical and true;
And remember, 'Patience, Patience,' is the watchword of a sage,
Not to-day nor yet to-morrow can complete a perfect age.</p>
        </main>
    );
}

export default Home;