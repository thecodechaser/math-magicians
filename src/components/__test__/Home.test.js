import { render, screen } from '@testing-library/react';
import Home from '../Home';

test('should contain p element ', () => {
  render(<Home />);
  const element = screen.getByRole('heading', { name: /welcome to our page/i });
  expect(element).toMatchSnapshot();
});
