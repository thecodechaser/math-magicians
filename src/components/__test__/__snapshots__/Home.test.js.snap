// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should contain p element  1`] = `
<h1
  class="home-heading"
>
  Welcome to our page!
</h1>
`;
